pkgs/top-level/perl-packages.nix

```diff
@@ -7727,17 +7727,43 @@ let self = _self // overrides; _self = with self; {
       description = "Embed a Perl interpreter in the Apache HTTP server";
     };
   };
-
   Mojolicious = buildPerlPackage {
-    name = "Mojolicious-4.63";
+    name = "Mojolicious-6.56";
     src = fetchurl {
-      url = mirror://cpan/authors/id/S/SR/SRI/Mojolicious-4.63.tar.gz;
-      sha256 = "f20f77e86fc560dac1c958e765ed64242dcf6343939ed605b45f2bbe2596d5e9";
+      url = mirror://cpan/authors/id/S/SR/SRI/Mojolicious-6.56.tar.gz;
+      sha256 = "82f73553836ac378edf825fd9f24be982653be9e0d78f8ba38b7841aabdafb02";
     };
+    propagatedBuildInputs = [ JSONPP ];
     meta = {
-      homepage = http://mojolicio.us;
+      homepage = http://mojolicious.org;
       description = "Real-time web framework";
-      license = stdenv.lib.licenses.artistic2;
+      license = "artistic_2";
+    };
+  };
+
+  MojoIOLoopForkCall = buildPerlModule {
+    name = "Mojo-IOLoop-ForkCall-0.17";
+    src = fetchurl {
+      url = mirror://cpan/authors/id/J/JB/JBERGER/Mojo-IOLoop-ForkCall-0.17.tar.gz;
+      sha256 = "886de5c3b44194a86228471075fac4036073bda19093e776c702aa65c3ef1824";
+    };
+    propagatedBuildInputs = [ IOPipely Mojolicious ];
+    meta = {
+      description = "Run blocking functions asynchronously by forking";
+      license = "perl";
```

Comment:
joachifm dixit (03/22 10:08):

> >        description = "Real-time web framework";
> > -      license = stdenv.lib.licenses.artistic2;
> > +      license = "artistic_2";
> > +    };
> > +  };
> > +
> > +  MojoIOLoopForkCall = buildPerlModule {
> > +    name = "Mojo-IOLoop-ForkCall-0.17";
> > +    src = fetchurl {
> > +      url = mirror://cpan/authors/id/J/JB/JBERGER/Mojo-IOLoop-ForkCall-0.17.tar.gz;
> > +      sha256 = "886de5c3b44194a86228471075fac4036073bda19093e776c702aa65c3ef1824";
> > +    };
> > +    propagatedBuildInputs = [ IOPipely Mojolicious ];
> > +    meta = {
> > +      description = "Run blocking functions asynchronously by forking";
> > +      license = "perl";
> 
> Is there not a `lib.licenses` equivalent for this? Perhaps add one if not.
> 

In the end this is artistic license:
http://dev.perl.org/licenses/

-- 

sg